client/html/src/Client/Html/Catalog/Product/Standard.php

```diff
@@ -0,0 +1,468 @@
+<?php
+
+/**
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ * @copyright Aimeos (aimeos.org), 2019-2019
+ * @package Client
+ * @subpackage Html
+ */
+
+
+namespace Aimeos\Client\Html\Catalog\Product;
+
+/**
+ * Implementation of catalog product section HTML clients for a configurable list of products.
+ *
+ * @package Client
+ * @subpackage Html
+ */
+class Standard
+	extends \Aimeos\Client\Html\Catalog\Base
+	implements \Aimeos\Client\Html\Common\Client\Factory\Iface
+{
+	private $subPartPath = 'client/html/catalog/product/standard/subparts';
```

Comment:
Can you please add documentation for this?

Reply:
Done